docs/markdown/reference/validation/README.md

```diff
@@ -86,3 +86,71 @@ This is a verbose example designed to show how Bootstrap-Vue and Vuelidate inter
 
 <!-- form-validation-1.vue -->
 ```
+
+## vee-validate
+
+[vee-validate](https://github.com/baianat/vee-validate) is a plugin for Vue.js that allows you to validate input fields and display errors. It has full support for `vue-i18n` and provides fairly good out of the box error messages.
+
+Same example as above just modified for vee-validate:
+
+```html
+<template>
+  <b-form @submit="onSubmit">
+    <b-form-group id="exampleInputGroup1"
+                  label="Name"
+                  label-for="exampleInput1">
+      <b-form-input id="exampleInput1"
+                    type="text"
+                    v-model="form.name"
+                    v-validate="{required: true, min:2}"
+                    :state="validateState('form.name')"
+                    aria-describedby="input1LiveFeedback"
+                    placeholder="Enter name" />
+      <b-form-invalid-feedback id="input1LiveFeedback">
+        This is a required field and must be at least 3 characters
+      </b-form-invalid-feedback>
+    </b-form-group>
+    <b-form-group id="exampleInputGroup2"
+                  label="Food"
+                  label-for="exampleInput2">
+      <b-form-select id="exampleInput2"
+                     :options="foods"
+                     v-validate="{required: true}"
+                     :state="validateState('form.foods')"
```

Comment:
Where this method from?

Reply:
Thanks for reviewing. I wrote it, but it didn't end up in the patch. I've updated the doc to include it.